extensions/wikia/CommunityHeader/CommunityHeaderController.class.php

```diff
@@ -1,25 +1,38 @@
 <?php
 
-use \Wikia\CommunityHeader\Sitename;
-use \Wikia\CommunityHeader\Wordmark;
-use \Wikia\CommunityHeader\Counter;
-use \Wikia\CommunityHeader\WikiButtons;
-use \Wikia\CommunityHeader\Navigation;
+use Wikia\CommunityHeader\Counter;
+use Wikia\CommunityHeader\Navigation;
+use Wikia\CommunityHeader\Sitename;
+use Wikia\CommunityHeader\WikiButtons;
+use Wikia\CommunityHeader\Wordmark;
 
 class CommunityHeaderController extends WikiaController {
```

Comment:
Don't we want to change that to WikiaService, or we handle that in another ticket?